airbyte-integrations/connectors/source-amazon-ads/acceptance-test-config.yml

```diff
@@ -47,4 +48,4 @@ tests:
       configured_catalog_path: "integration_tests/configured_catalog_report.json"
       ignored_fields:
         "sponsored_products_report_stream": ["updatedAt"]
-      timeout_seconds: 3600
```

Comment:
The new line at the end of the file is missing. Run `./gradlew format` in the project root directory to fix code formatting issues across all files.

Reply:
I think this is the old file content, please look at the new content where newline has been added at the end.

Reply:
Anyway, there are still formatting issues in spec.json. Please run `./gradlew format` to ensure all code formatting is correct.

Reply:
I have run it in the past and ran again,it doesn't flag any formatting issues

Reply:
alright, this time it did flagged